### PR TITLE
[Spotless] Applying Google Code Format for legacy directory (pt 2/4) #20

### PR DESCRIPTION
**Pull request by MitchellGale**
### Description
Applies spotless for part of legacy directory in project. Sets checkstyle to not fail on failure for legacy directory.
 
### Issues Resolved
https://github.com/opensearch-project/sql/issues/1101
 
### Check List
- [x] New functionality includes testing.
  - [x] All tests pass, including unit test, integration test and doctest
- [x] New functionality has been documented.
  - [x] New functionality has javadoc added
  - [x] New functionality has user manual doc added
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by acarbonetto — DISMISSED** [review]

**Comment by Yury-Fridlyand** [comment]
Compilation fails

**Review by acarbonetto — DISMISSED** [review]

File: opensearch/src/main/java/org/opensearch/sql/opensearch/storage/script/filter/lucene/LuceneQuery.java [hunk 1]
```diff
@@ -224,6 +224,15 @@ private ExprValue cast(FunctionExpression castFunction) {
                   return new ExprTimeValue(expr.valueOf().timeValue());
                 }
               })
+          .put(
```
Comment (Yury-Fridlyand) [inline opensearch/src/main/java/org/opensearch/sql/opensearch/storage/script/filter/lucene/LuceneQuery.java:227]: revert

Comment (MitchellGale) [reply opensearch/src/main/java/org/opensearch/sql/opensearch/storage/script/filter/lucene/LuceneQuery.java:227]: Reverted

**Review by acarbonetto — APPROVED** [review]

**Review by Yury-Fridlyand — APPROVED** [review]

**Comment by joshuali925** [comment]
ported by #2016, removing backport failed label

**Comment by MitchellGale** [comment]
Thank you

**Pull request merged**